functions/prepaped-check-sync/message-schema/message.v1.json

```diff
@@ -0,0 +1,78 @@
+{
```

Comment:
is it the responsibility of the message sender to provide the updated payload?  or should it just indicate the checkCode of the check that needs resyncing?  the latter seems feels like a better choice

Reply:
@GuyHarwood I would think the prepared-check-sync function will fetch the relevant `PartitionKey`, `RowKey `,  and `config` then update the latter with the new settings and push to table storage?
In that case the message would not need the checkCode but only these 3 properties that I have highlighted.

Thoughts? 🤔